app/models/review.rb

```diff
@@ -78,7 +78,9 @@ def description(heading)
 
   def fields_for_heading(heading)
     if questions[heading].present?
-      questions[heading].fields
+      questions[heading].fields.reject { |field|
+        ['project_worked_on', 'role_description'].include?(field)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%w` or `%W` for an array of words.